pyresample/geometry.py

```diff
@@ -1026,6 +1087,31 @@ def freeze(self, lonslats=None, resolution=None, shape=None, proj_info=None):
           the shape of the resulting area.
         proj_info:
           complementing parameters to the projection info.
+        antimeridian_mode:
+            How to handle lon/lat data crossing the anti-meridian of the
+            projection. This currently only effects lon/lat geographic
+            projections and data cases not covering the north or south pole.
+            The possible options are:
+
+            * modify_extents: Set the X bounds to the edges of the data, but
```

Comment:
Should these parameter names have quotes `"` around them?

Reply:
Sure.